.travis.yml

```diff
@@ -11,16 +12,24 @@ services:
   - postgresql
 before_install:
   - rvm install ruby-1.9.3-p551
+  # Repo for Yarn
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
+  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -y -qq yarn=$YARN_VERSION
```

Comment:
This [Travis blog post seems](https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn) to suggest that Yarn is pre-installed for you if you have a `yarn.lock`